src/main/java/com/google/codeu/data/Datastore.java

```diff
@@ -161,4 +161,21 @@ public int getTotalMessageCount() {
     return results.countEntities(FetchOptions.Builder.withDefaults());
   }
 
+  /**
+   * This is to create an user form data to put in the datastore
+   * TODO: Replace getUser() with UUID and use userChart.getId().toString()
+   */
+  public void storeUserFormData(Tea userChart) {
+    
+    Entity userFormData = new Entity("UserFormData", userChart.getUserName());
```

Comment:
This entity name seems off to me. I would expect it more to be like `UserTeaConsumption`, `TeaConsumptionByUser`, or `TeaConsumption`. That there is no mention of tea at all is surprising.

Reply:
Fixed the entity name from  ```userFormData``` -> ```userTeaConsumption```

I agree, it was weird for the name to not mention anything about the tea as it is taking/holding the user's tea data.